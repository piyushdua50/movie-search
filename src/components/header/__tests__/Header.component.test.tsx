import React from "react";
import { BrowserRouter, Routes, Route } from "react-router-dom";
import { render } from "@testing-library/react";
import Header from "../Header.component";
test("render the Header correctly", () => {
  const { container } = render(
    <BrowserRouter>
      <Routes>
        <Route path="/" element={<Header />} />
      </Routes>
    </BrowserRouter>
  );
  expect(container).toMatchSnapshot();
});
