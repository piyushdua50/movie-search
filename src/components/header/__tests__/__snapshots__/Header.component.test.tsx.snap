// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the Header correctly 1`] = `
<div>
  <div
    class="headerContainer"
  >
    <h1
      class="headerTitle"
    >
      Home
    </h1>
    <h1
      class="headerTitle"
    >
      Favorites
    </h1>
  </div>
</div>
`;
